.editorconfig

```diff
@@ -161,7 +161,7 @@ csharp_space_between_square_brackets = false
 dotnet_code_quality.api_surface = all
 
 # License header
-file_header_template = Copyright (c) .NET Foundation and contributors. All rights reserved.\nLicensed under the MIT license. See LICENSE file in the project root for full license information.\n
+file_header_template = Copyright (c) .NET Foundation and contributors. All rights reserved.\nLicensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I see that the diagnostic is marked as a warning: https://github.com/dotnet/sdk/blob/dfae9b7b85f2b693a9a55f8d70b08af01d3ee83f/.editorconfig#L335-L336

Why isn't this enforced on CI? Or are none of the code style rules enforced on build on this repo?

Reply:
Not sure. Roslyn's model of running a separate "correctness" CI leg would be nice, but I don't think this repo has it.